docs/_resources/contributors.md

```diff
@@ -0,0 +1,124 @@
+
+1. 🐛 **Working on existing bugs:** You can assist by identifying and addressing existing bugs listed in our repository. 
+2. 🛠️ **Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool:** This involves proposing new features, fixing bugs, or suggesting changes for existing tools. 
+3. 💡 **Submitting a proposal for a new tool:** If you've developed a FinOps-related tool and want it to be part of the FinOps Toolkit, you can submit a proposal for consideration.
+4. 👀 **Code review:** Code review is a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
+5. 💬 **Answering questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+6. 📚 **Documentation update:** Documentation serves as a crucial resource for users, developers, and contributors alike. Keeping documentation accurate, comprehensive, and up-to-date is essential for the success of the toolkit.
+
+We appreciate your interest in contributing to the FinOps Toolkit project!
+
+### 🐛 1. Working on existing bugs
+
+If you're eager to contribute but unsure where to begin, a great starting point is to review the list of existing feedback and identify issues that align with your skills and interests. 
+
+1. Look through the list of existing bugs in the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues).
+2. Assign yourself to the bug you want to work on.
+
+If this is the first time you are working on the FinOps Toolkit repository, make sure to review our [Get Started Wiki.](https://github.com/microsoft/finops-toolkit/wiki#-test-and-verify)
+
+Once you are assigned to the bug, GitHub will automatically assign the tag "Community Contributor" to inform the team responsible for that tool of your contribution. 
+
+### 🛠️ 2. Suggesting and owning a new feature, fixing an existing bug, or suggesting a change for an existing tool
+
+#### Submitting a New Issue and Willing to Fix
+
+If you have a new feature suggestion, identified a bug, or have ideas for improving existing tools, we encourage you to submit an issue. Your feedback helps us ensure that the toolkit meets the needs of its users. And if you're willing to take it a step further and contribute to implementing your suggestions, we welcome your contributions with open arms! 
+
+To do that, follow these steps: 
+
+1. Go to the [Issues tab](https://github.com/finopsfoundation/FinOpsToolkit/issues) and click on "New issue".
+2. Choose the option that best describes your contribution (bug fix, change request, or feature request).
+3. Check the box "Willing to fix" at the bottom of the page.

```

Comment:
If they're willing to fix it, should we just ask them to submit a PR directly? I'm thinking we just remove this whole "willing to fix" part.

Reply:
Maybe they are willing to fix but they are not so comfortable working with GitHub (branches can be scary 👻) or they want to fix it, but are unsure how to proceed, etc. The idea of this checkbox is just a way for us to know that there is a person trying to fix that, and we may want to reach out to them and see if they need any help with it. 
Also, maybe there is value on reporting how many people are trying to fix these issues.